x/merkledb/trie_test.go

```diff
@@ -792,13 +789,13 @@ func Test_Trie_MultipleStates(t *testing.T) {
 
 			initialSet := 1000
 			// Populate initial set of keys
-			root, err := db.NewView()
+			root, err := db.NewView(nil)
 			require.NoError(err)
 			kv := [][]byte{}
 			for i := 0; i < initialSet; i++ {
 				k := []byte(strconv.Itoa(i))
 				kv = append(kv, k)
-				require.NoError(root.Insert(context.Background(), k, hashing.ComputeHash256(k)))
+				require.NoError(root.(*trieView).insert(k, hashing.ComputeHash256(k)))
```

Comment:
nit: seems we can do this without the cast, building an `ops`  array then calling db.NewView